proposals/field-keyword-nullability.md

```diff
@@ -33,8 +44,8 @@ The nullability of the backing field is determined as follows:
 - If the field has nullability attributes such as `[field: MaybeNull]`, `AllowNull`, `NotNull`, or `DisallowNull`, then the field's nullable annotation is the same as the property's nullable annotation.
```

Comment:
> Then, if nullable analysis of the getter yields no nullable warnings, the property is *null-resilient*. Otherwise, it is not *null-resilient*.

Could read this two ways:

1. The analysis of the getter produces no warnings at all 
2. The analysis of the getter results in no warnings after considering `!`, `#pragma warning disable`, etc ...

Guessing it's (2)

Sorry: the PR won't let me put this comment on the appropriate line

Reply:
@jaredpar yes.  it's '2'.  The user putting a suppression is them saying "tehre's actually no null-issue here" which we'll continue to trust wrt to trying to determine the null-safety of the code :)